encoding/_toml/parser.ts

```diff
@@ -0,0 +1,897 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import { deepAssign } from "https://deno.land/std@0.101.0/_util/deep_assign.ts";
```

Comment:
Please use relative paths instead of versioned std module url (to prevent multiple versions of std mixed in the future)

Reply:
Oh, I didn't notice that. Thank you. I fixed!